app/src/main/java/io/neurolab/SplashActivity.java

```diff
@@ -0,0 +1,23 @@
+package io.neurolab;
+
+import android.content.Intent;
+import android.os.Build;
+import android.os.Handler;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class SplashActivity extends AppCompatActivity {
+
+    private static final long SPLASH_SCREEN_DELAYED_TIME = 3000;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_splash);
+
+        new Handler().postDelayed(() -> {
+            startActivity(new Intent(SplashActivity.this, MainActivity.class));
+            finish();
+        }, SPLASH_SCREEN_DELAYED_TIME);
```

Comment:
Horrible idea. Why should I wait 3 seconds more to open the app?

Reply:
Sorry I will change it to 1 second.